adding trailing separators into rlistings

Melkiades opened the pull request:
Fixes #237

shajoezhu reviewed (changes requested):
thanks for the PR @Melkiades , can we have a snapshot test to track this. thanks!

Melkiades commented on tests/testthat/test-print.R:
```diff
@@ -180,3 +180,76 @@ testthat::test_that("sas rounding support", {
+  result <- head(result, 15)
+
+  expect_equal(
+    sum(
+      sapply(
+        strsplit(toString(result), "\n")[[1]],
+        function(x) {
+          x == paste0(rep(substr(x, 1, 1), nchar(x)), collapse = "")
+        }, USE.NAMES = FALSE
+      )
+    ),
+    9
```
This says the number of lines that repeat themselves are 9 ( 8 + 1 that is the standard separator)

Melkiades commented:
> thanks for the PR @Melkiades , can we have a snapshot test to track this. thanks!

@shajoezhu thanks! added ;)

shajoezhu commented:
> > thanks for the PR @Melkiades , can we have a snapshot test to track this. thanks!
> 
> @shajoezhu thanks! added ;)

Thanks @Melkiades !
can you make a longer one please? for example, it would be good to mix it with pagination as well, and see if they both can work. thanks!

Melkiades commented:
> > > thanks for the PR @Melkiades , can we have a snapshot test to track this. thanks!
> > 
> > 
> > @shajoezhu thanks! added ;)
> 
> Thanks @Melkiades ! can you make a longer one please? for example, it would be good to mix it with pagination as well, and see if they both can work. thanks!

Done! There were issues indeed with pagination. Fixed and added tests ;)

shajoezhu reviewed (approved):
very powerful feature! Thanks a lot @Melkiades 

lgtm!
